src/netex_idf/lines.rs

```diff
@@ -0,0 +1,169 @@
+// Copyright 2017 Kisio Digital and/or its affiliates.
+//
+// This program is free software: you can redistribute it and/or
+// modify it under the terms of the GNU General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+
+use super::EUROPE_PARIS_TIMEZONE;
+use crate::{
+    minidom_utils::{TryAttribute, TryOnlyChild},
+    model::Collections,
+    objects::{Company, Network},
+    Result,
+};
+use failure::{format_err, ResultExt};
+use log::info;
+use minidom::Element;
+use std::{fs::File, io::Read};
+use transit_model_collection::CollectionWithId;
+
+fn load_networks_companies(
```

Comment:
you could split later in 2 functions `load_network` and `load_company`

Reply:
Ok. We'll see when we get the complete (and heavier) data set. For the moment we are staying on a loop for the 2